web/modules/custom/dpl_fees/src/Form/FeesListSettingsForm.php

```diff
@@ -63,28 +63,43 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
+      '#title' => $this->t('Basic settings', [], ['context' => 'Fees list settings form']),
       '#tree' => FALSE,
     ];
 
     $form['settings']['fees_and_replacement_costs_url'] = [
-      '#type' => 'url',
-      '#title' => $this->t('Fees and Replacement costs URL'),
-      '#description' => $this->t('File or URL containing the fees and replacement costs'),
+      '#type' => 'linkit',
+      '#title' => $this->t('Fees and Replacement costs URL', [], ['context' => 'Fees list settings form']),
+      '#description' => $this->t('File or URL containing the fees and replacement costs. <br>
+                                         You can add a relative url (e.g. /takster). <br>
+                                         You can search for an internal url. <br>
+                                         You can add an external url (starting with "http://" or "https://").', [], ['context' => 'Fees list settings form']),
+      '#autocomplete_route_name' => 'linkit.autocomplete',
+      '#autocomplete_route_parameters' => [
+        'linkit_profile_id' => 'default',
+      ],
       '#default_value' => $config->get('fees_and_replacement_costs_url') ?? DplFeesSettings::FEES_AND_REPLACEMENT_COSTS_URL,
     ];
 
     $form['settings']['payment_overview_url'] = [
-      '#type' => 'url',
-      '#title' => $this->t('Payment overview url'),
+      '#type' => 'linkit',
+      '#title' => $this->t('Payment overview url', [], ['context' => 'Fees list settings form']),
+      '#description' => $this->t('URL containing the payment overview. <br>
+                                         You can add a relative url (e.g. /takster). <br>
+                                         You can search for an internal url. <br>
+                                         You can add an external url (starting with "http://" or "https://")', [], ['context' => 'Fees list settings form']),
+      '#autocomplete_route_name' => 'linkit.autocomplete',
+      '#autocomplete_route_parameters' => [
+        'linkit_profile_id' => 'default',
+      ],
       '#default_value' => $config->get('payment_overview_url') ?? DplFeesSettings::PAYMENT_OVERVIEW_URL,
     ];
 
     $form['settings']['fee_list_body_text'] = [
       '#type' => 'textarea',
-      '#title' => $this->t('Intro text'),
-      '#description' => $this->t('Display an intro-text below the headline'),
-      '#default_value' => $config->get('fee_list_body_text') ?? $this->t('Fees and replacement costs are handled through the new system "Mit betalingsoverblik.'),
+      '#title' => $this->t('Intro text', [], ['context' => 'Fees list settings form']),
+      '#description' => $this->t('Display an intro-text below the headline', [], ['context' => 'Fees list settings form']),
+      '#default_value' => $config->get('fee_list_body_text') ?? $this->t('Fees and replacement costs are handled through the new system "Mit betalingsoverblik.', [], ['context' => 'Fees list settings form']),
```

Comment:
didnt we create methods for getting fee settings like this?

Reply:
Yes, we have implemented some of these methods, but not throughout the whole project yet. I have added 2 new methods for getting fee settings now.